### DISCUSSION TITLE
How could one make an app.

### DISCUSSION THREAD

**Discussion by Artimidorus** [victordiaz/PHONK#135] [Q&A] [answered]
Using PHONK, is it possible to make an app that doesn't require the user to install PHONK?
If this is already possible how can I do it?

**Comment by victordiaz** [comment, marked as answer]
Yes, its kinda possible but not very straightforward.
Give me 1 or 2 weeks so things will be more clear on my side and I will come back with some guide :)

**Reply by Artimidorus** [reply]
Ok thank you for the response. I look forward to using this to publish my own apps.

**Reply by Artimidorus** [reply]
Is there a way to get email updates when new documentation becomes available?

**Comment by S1700** [comment]
Hello everyone, sorry to revive this post however I just wanted to know if this guide has been made, or if anyone else knows how one could make an application. Thanks!

**Reply by Artimidorus** [reply]
If a guide has been made I've yet to find it.  😔

**Reply by S1700** [reply]
I'll try contacting Victor and see if I can make it for him. But from what i've seen its been done.

**Reply by S1700** [reply]
> But from what i've seen its been done.

The apk not the guide lol
